server/controllers/middlewares/auth/protectedRoute.js

```diff
@@ -0,0 +1,14 @@
+const jwt = require('jsonwebtoken');
+
+const protectedRoute = async (req, res, next) => {
+  jwt.verify(req.cookies.token, process.env.SECRET_KEY, (err, token) => {
+    if (err) {
+      res.status(401).json({ statusCode: 401, message: 'Un-Authorized' });
+    } else {
+      req.user = token;
```

Comment:
I think there is no need to pass the token in request body to next middleware, as I see you don't use it.
I think we need it in case we need to take information stored in token as user id for example.